wrappers/emerge-wrapper

```diff
@@ -107,6 +106,8 @@ cross_wrap_etc()
 	EOF
 	fi
 
+	PORTAGE_CONFIGROOT=${SYSROOT} eselect profile set --force ${PROFILE}
```

Comment:
I was going to suggest setting `ROOT=/usr/${CHOST}` and `EPREFIX=` instead, but looking at how profile.eselect works, I think what you have is correct because the repos are not configured under `/usr/${CHOST}/etc/portage`.

It's slightly brittle in that it relies on profile.eselect calling `portageq get_repo_path "${EROOT:-/}"` and `portageq get_repo_path` (thankfully) ignoring `PORTAGE_CONFIGROOT`, but that's what we have to work with.

But it is probably best to quote `PROFILE` here.

```suggestion
	PORTAGE_CONFIGROOT=${SYSROOT} eselect profile set --force "${PROFILE}"
```

Reply:
As discussed, I'm handling these changes now, but for the record, I'm going to move this line to `cross_wrap` so that it runs unconditionally. I think `--force` makes sense, but it's ineffective with the line where it is.